x-pack/filebeat/module/ibmmq/_meta/kibana/7/dashboard/Filebeat-IBMMQ-Overview.json

```diff
@@ -386,7 +386,7 @@
             },
             "totalFunc": "sum"
           },
-          "title": "[Filebeat] Top 5 Errors",
+          "title": "[Filebeat IBM MQ] Top 5 Errors",
```

Comment:
sorry Im confused a bit, why this title and and the title in Ln 295 has different format?

Reply:
Umm, this title is for a table, not a visualization or dashboard, so this is not checked yet, I guess I changed it with some replace :grimacing: let me use the same format as visualizations by now.

Reply:
Great thanks!!